src/processing_steps/0200_generate_byte_hdf5.py

```diff
@@ -71,6 +71,8 @@
 
     # Store metadata in both files for each subvolume scan
     for h5file in [h5file_msb,h5file_lsb]:
+        # never a novisim tomogram at this point, but added to all *.h5 files for later checks
```

Comment:
A bit funny wording, but I get what you're saying.

Reply:
I agree. This has been rephrased. Is it any better?